lib/mastodon/cli/accounts.rb

```diff
@@ -6,6 +6,7 @@
 module Mastodon::CLI
   class Accounts < Base
     option :all, type: :boolean
+    option :verbose, type: :boolean, aliases: [:v]
```

Comment:
I don't think its universal, but when I think of `-v` for a CLI, it's usually a "print version" shorthand

Reply:
Yes, usually is. I'm matching pre-existing style from elsewhere in the CLI here.

Reply:
Also just to clarify here ... at the CLI top-level this command -- `bin/tootctl version` or `bin/tootctl -v` -- does what you think (returns version). The `-v` alias used here is used in sub-commands as a verbose alias (ie, `bin/tootctl cache clear --verbose` or whatever).